### PR TITLE
Login Module fix

### PR DESCRIPTION
**Pull request by justinma246**
https://github.com/kaazing/gateway/pull/604

### CODE REVIEW

**Comment by justinma246** [comment]
@jitsni

**Comment by jitsni** [comment]
travis build is failing and seems to be related this changeset.

**Comment by justinma246** [comment]
@jitsni I made some changes, thanks for the heads up!

File: server/src/main/java/org/kaazing/gateway/server/config/parse/translate/June2016Validator.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import org.kaazing.gateway.server.config.parse.translate.june2016.RemoveRequireUser;
+
+public class June2016Validator extends GatewayConfigTranslatorPipeline {
+
+    public June2016Validator() {
+        super();
+        
+        // Set the namespace
+        addTranslator(new RemoveRequireUser());
```
Comment (dpwspoon) [inline server/src/main/java/org/kaazing/gateway/server/config/parse/translate/June2016Validator.java]: I don't get this design reason... Why are removing require user in the 2016 Validator

Comment (dpwspoon) [inline server/src/main/java/org/kaazing/gateway/server/config/parse/translate/June2016Validator.java]: We should be removing the realm on the old namespace only if there is a realm and no authorization constraint.  Meaning the translator is wrong and this should be added to a different Translator Pipeline

**Pull request merged**